docs/marketplace/dev/architecture-overview/202108.0/marketplace-merchant-portal-architecture-overview.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Merchant Portal Architecture overview
+description:
+template: concept-topic-template
+---
+This document depicts how a Spryker Marketplace MerchantPortal is designed in general. 
+
+The following diagrams outlines the relation between Zed, MerchantPortal, Backoffice and DB. Learn more about [Marketplace MerchantPortal Core feature here](/docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-merchant-portal-core-feature-walkthrough/marketplace-merchant-portal-core-feature-walkthrough.html).
+
+![MerchantPortal Architecture overview](https://confluence-connect.gliffy.net/embed/image/4b06167a-3c9a-483c-8b57-32544b211fc5.png?utm_medium=live&utm_source=custom)
+
+### Zed and MerchantPortal
```

Comment:
they should not be mixed in one line, 
Zed is an application layer and
MePO is an application based on zed.